### DISCUSSION TITLE
How to calculate kij through PC-SAFT in Clapeyron？

### DISCUSSION THREAD

**Discussion by Chaunceywei** [ClapeyronThermo/Clapeyron.jl#235] [Q&A] [answered]
How to calculate kij through PC-SAFT in Clapeyron？

**Comment by longemen3000** [comment, marked as answer]
do you mean estimate PC-SAFT kij values, given equilibrium data?

**Reply by Chaunceywei** [reply]
yes, by fitting experimental data to obtain kij

**Reply by pw0908** [reply]
You can take a look at the example notebooks here: https://github.com/ClapeyronThermo/Clapeyron.jl/blob/master/examples/parameter_estimation.ipynb

The relevant example will be that of NRTL where, rather than fitting kij, youll be fitting the unlike epsilon parameter (epsilon_ij). This parameter is also symmetric, so no need to worry about asymmetries.

Hope this helps!

**Reply by Chaunceywei** [reply]
**Thanks for your reminder! this helped me a lot**
